File: vcx/wrappers/python3/vcx/state.py
from enum import IntEnum


class State(IntEnum):
    Undefined = 0,
    Initialized = 1,
    OfferSent = 2,
    RequestReceived = 3,
    Accepted = 4,
    Unfulfilled = 5,
    Expired = 6,
    Revoked = 7,
    Redirected = 8,
    Rejected = 9,


class ProofState(IntEnum):
    Undefined = 0,
    Verified = 1,
    Invalid = 2


class PublicEntityState(IntEnum):
    Built = 0,
    Published = 1

File: vcx/wrappers/java/demo/src/main/java/utils/State.java
package utils;

public class State {
    public static class StateType {
        public static final int None = 0;
        public static final int Initialized = 1;
        public static final int OfferSent = 2;
        public static final int RequestReceived = 3;
        public static final int Accepted = 4;
        public static final int Unfulfilled = 5;
        public static final int Expired = 6;
        public static final int Revoked = 7;
        public static final int Redirected = 8;
        public static final int Rejected = 9;
    }

    public static class ProofState {
        public static final int Undefined = 0;
        public static final int Verified = 1;
        public static final int Invalid = 2;
    }
}
